.gitignore

```diff
@@ -1,6 +1,5 @@
 carpenter
 .idea/
-fixtures/
```

Comment:
This should have the `fixtures/` directory

Reply:
this is missing from the upstream repo. I wanted to confirm before I added it to my PR